src/applications/disability-benefits/all-claims/tests/prefill-transformer.unit.spec.js

```diff
@@ -0,0 +1,247 @@
+import { expect } from 'chai';
+import prefillTransformer, {
+  filterServiceConnected,
+  addNoneDisabilityActionType,
+} from '../prefill-transformer';
+
+import {
+  SERVICE_CONNECTION_TYPES,
+  disabilityActionTypes,
+} from '../../all-claims/constants';
+
+describe('526v2 prefill transformer', () => {
+  const noTransformData = {
+    metadata: { test: 'Test Metadata' },
+    formData: { testData: `This isn't getting transformed` },
+    pages: { testPage: 'Page 1' },
+  };
+
+  it('should return a copy of the prefill data', () => {
+    const { pages, formData, metadata } = noTransformData;
+    const transformedPrefill = prefillTransformer(pages, formData, metadata);
+    expect(transformedPrefill).to.not.equal(noTransformData);
+    expect(transformedPrefill).to.deep.equal(noTransformData);
```

Comment:
Heh, this is misleading. :sweat_smile: For a second there, I was like, "Wait, should it equal or should it not? *Make up my mind!*"

Reply:
Yeah, I thought so as well. Probably means I should come up with something better.